packages/react-admin-core/src/table/useTableQuerySort.tsx

```diff
@@ -2,8 +2,8 @@ import * as React from "react";
 import { usePersistedState } from "./usePersistedState";
 
 export enum SortDirection {
-    ASC = "asc",
-    DESC = "desc",
+    ASC = "ASC",
+    DESC = "DESC",
```

Comment:
macht diese änderung wo inkompatibel was kaputt?

Und warum überhaupt brauchst du das anders?

Reply:
weil ein enum im graphql immer so ausschaut:
enum SortDirection {
  ASC
  DESC
}

und wenn ich dann den value aus sortApi übergebe kakt typescript immer ab weil das inkompatibel ist

Reply:
Ich fürchte, das ist eine inkompatible Änderung...

Reply:
in graphql haben wir halt problem weil ich dann bei jedem sortApi aufruf einen fehler bekomm und ich will das nicht bei jedem mit einer eigenen funktion fixen müssen
was macht es hin wen der string jetzt in großbuchstaben daher kommt? is ja das gleiche wort?

Reply:
Naja. wann der parameter übergeben wird (so wie du es tun willst) ist er halt dann anders geschrieben.

Aber wenn das so sinn macht (was du ja jetzt erklärt hast) ist es halt eine inkompatible änderung. eine nachricht in react-admin-dev channel und fertig :)